code/modules/hydroponics/plant_genes.dm

```diff
@@ -521,6 +522,12 @@
 			HY.update_icon()
 			HY.visible_message("<span class='warning'>The [H.myseed.plantname] spreads!</span>")
 
+// It boosts chemical output of a plant by rate
+/datum/plant_gene/trait/richer_juice
+	name = "Richer Juice"
+	rate = 2
+	trait_id = "chem_boots"
```

Comment:
```suggestion
	trait_id = "chem_boost"
```

Again, this just seems like a typo - just highlighting the other for easy access.